mito-ai/src/Extensions/AiChat/ChatMessage/ChatDropdown.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { ExpandedVariable } from './ChatInput';
+
+interface ChatDropdownProps {
+    options: ExpandedVariable[];
+    selectedIndex: number;
+    onSelect: (variableName: string, parentDf?: string) => void;
+}
+
+const ChatDropdown: React.FC<ChatDropdownProps> = ({
```

Comment:
I don't think we need to do this right now since we don't yet have another use for the dropdown. But just wanted to call this out.

We've had a lot of success building component libraries for ourselves in the past. In the mitosheet/src/mito/components folder you can see a bunch of components we've created. Having these general purpose components speeds up development time so so much. So I think we should treat this dropdown as a component that we might want to reuse across the mito-ai extension. 

Creating good components means we should: 
1. Have good scoping of what the component is responsible for and what the parent of the component is responsible for. The parent component should be responsible for passing in the data, placing the component in the right spot, etc. The component itself should be responsible for its own logic. 
2. Variable names that make sense in the context of the component regardless of where it is used in the codebase. So for example, instead of the options being typed `ExpandedVariable[]`, we could type it `DropdownOptions[]` and then make sure that the things we pass in fit that.